.changeset/unlucky-walls-tie.md

```diff
@@ -1,5 +1,7 @@
 ---
-"@coinbase/onchainkit": minor
+"@coinbase/onchainkit": patch
 ---
 
-**feat**: Swap success state - refetch balances and clear inputs by @0xAlec #1089
+- **feat**: added connect wallet functionality to Swap component for disconnected users. By @abcrane123 #1173
+- **fix**: refetched balances and cleared inputs after Swap succeeds. By @0xAlec #1089
```

Comment:
nit but i feel like it would make more sense to have `added logic to refetch balances and clear inputs after swap succeeds`

Reply:
changed